vulcan/lib/client/jsx/components/workflow/workflow_manager.tsx

```diff
@@ -59,20 +59,20 @@ export default function WorkflowManager({
     (figureId: number, localSession: VulcanFigureSession | null) => {
       showErrors(
         fetchFigure(projectName, figureId).then((figureResponse) => {
-          let fromDatabase = true;
+          let useLocal = true;
 
           if (
             localSession &&
             !_.isEqual(localSession.inputs, figureResponse.inputs)
           ) {
-            fromDatabase = confirm(
-              'You have an edited, local version of that figure. Discard it?'
+            useLocal = confirm(
+              'You have an edited, local version of that figure. Click OK to use your local version, Cancel to discard it.'
```

Comment:
Could we beef this up a bit?  I think it'd be nice to be explicit that unsaved changes will be discarded if the user cancels.  No getting them back without recreating the work, which reminds me of how video games always try to be explicit when a (re)load might lose unsaved progress.

Maybe:
"Your local version of this figure diverges from the last saved version. Do you want to keep your changes?  Click 'OK' to use your local version, or 'Cancel' to discard all unsaved changes and load the last saved state."

Reply:
Edited for verbosity: "You have unsaved changes. Click 'OK', etc."

Reply:
So "You have unsaved changes. Click 'OK',  to use your local version, or 'Cancel' to discard all unsaved changes and load the last saved state.", just trimming the beginning of my edit?  :+1:  I like that.

Reply:
Heh, slightly contradictory with being more explicit? @dtm2451 , does cutting down the first half but leaving the explicit wording in the OK / Cancel statements provide enough detail?

Reply:
I think so!